packages/components/src/components/Menu/Menu.test.tsx

```diff
@@ -36,4 +36,21 @@ describe("<Menu />", () => {
     expect(screen.getByText("Item 1")).toBeInTheDocument();
     expect(screen.getByText("Item 2")).toBeInTheDocument();
   });
+
+  it("allows a disabled prop for a menu item and disables the button", () => {
+    render(
+      <Menu items={[{ onClick: jest.fn(), label: "Item 1", disabled: true }]} />
+    );
+
+    const menuButton = screen.getByRole("button");
+    expect(menuButton).toBeInTheDocument();
+
+    userEvent.click(menuButton);
```

Comment:
I believe that we need an await here, right?

Reply:
Yeah, we need it. Do you guys think we should have this as a lint error? 🤔 
I've seen more cases like this.

Reply:
Anyway, I can start the discussion in the next Pluto's meeting. 🙏🏻